Common/Orders/OrderEvent.cs

```diff
@@ -33,11 +33,21 @@ public class OrderEvent
         public Symbol Symbol;
 
         /// <summary>
+        /// The date and time of this event (UTC).
+        /// </summary>
+        public DateTime Time;
+
+        /// <summary>
```

Comment:
Also, we need to verify this is being set by everyone who creates this. Recommend making it a required constructor parameter. I'm mostly thinking of the brokerage implementation who don't know to set this value.